src/index.njk.md

```diff
@@ -508,6 +508,15 @@ Here's a list of plugins that are provided by the awesome Metalsmith community.
 
 If you write your own plugin, submit a pull request to the [metalsmith.io](https://github.com/metalsmith/metalsmith.io/blob/master/lib/data/plugins.json) repository and it will show up here!
 
+In the pull request you can add the following properties to your plugin:
+```js
+"name": "<package-name>", // title on https://metalsmith.io/ (also for filtering)
+"icon": "link" | "files" ... // icon next to the title,
```

Comment:
@webketje if there are more properties which can be used or you have something like a schema for it I can include this as well.